test/cli/test_cli.py

```diff
@@ -135,6 +177,51 @@ def test_should_not_find_non_existing_workflow(self):
         self.assertEqual(exception_message[:len(expected_prefix)], expected_prefix)
         self.assertEqual(exception_message[-len(expected_suffix):], expected_suffix)
 
+    def test_verify_workflow_should_pass_for_existing_workflows_with_specified_id(self):
+        # when
+        verify_workflow('ID_1', ['test_module', 'other_test_module'], project_dir=TEST_PROJECT_DIR)
+        verify_workflow('ID_9', ['test_module', 'other_test_module'], project_dir=TEST_PROJECT_DIR)
+
+        # then
+        # no error is raised
+
+    def test_verify_workflow_should_pass_for_existing_workflows_without_specified_id(self):
+        # when
+        verify_workflow('ALL', ['test_module', 'other_test_module'], project_dir=TEST_PROJECT_DIR)
+
+        # then
+        # no error is raised
+
+    def test_verify_workflow_should_raise_error_for_non_existing_workflow(self):
+        # then
+        with self.assertRaises(Exception):
+            # when
+            verify_workflow('I DONT EXIST', ['test_module', 'other_test_module'], project_dir=TEST_PROJECT_DIR)
+
+    def test_verify_workflow_should_raise_error_for_broken_workflow_without_specified_id(self):
+        # then
+        with self.assertRaises(Exception):
+            # when
+            verify_workflow('ALL', ['broken_workflows'], project_dir=TEST_PROJECT_DIR)
+
+    def test_verify_workflow_should_raise_error_for_workflow_throwing_exception(self):
+        # then
+        with self.assertRaises(Exception):
+            # when
+            verify_workflow('ID_999', ['broken_workflows'], project_dir=TEST_PROJECT_DIR)
+
+    def test_verify_workflow_should_raise_error_for_workflow_with_syntax_error(self):
+        # then
+        with self.assertRaises(Exception):
+            # when
+            verify_workflow('ID_998', ['broken_workflows'], project_dir=TEST_PROJECT_DIR)
```

Comment:
Now we fail the whole build when there are any import issue (this include wrong import, global-thrown exception etc).  So basically all tests with `broken_workflows` do the same thing - fail at the first wrong worfklow.

In any case I don't see a big reasons to add those tests (as this is already covered).

Reply:
I wrote them for my own purpose, actually, to confirm that it fails uniformly for different kinds of problems.

Reply:
Finally decided to drop these tests.